Src/VimCore/WordUtil.fs

```diff
@@ -62,34 +134,90 @@ type internal WordUtil() =
             else 
                 None
 
-    /// Create an ITextStructure navigator for the ITextBuffer where the GetWordExtent function 
-    /// considers word values of the given WordKind.  Use the base ITextStructureNavigator of the
-    /// ITextBuffer for the rest of the functions
-    ///
-    /// Note: This interface can be invoked from any thread via ITextSearhService
-    [<UsedInBackgroundThread()>]
-    static member CreateTextStructureNavigator wordKind contentType = 
+/// ITextStructure navigator where the GetWordExtent function uses a fixed definition
+/// of word
+[<UsedInBackgroundThread>]
+[<Class>]
+[<Sealed>]
+type SnapshotWordNavigator
+    (
+        _wordUtil: SnapshotWordUtil,
+        _contentType: IContentType
+    ) =
+
+    member x.WordUtil = _wordUtil
+    member x.ContentType = _contentType
+
+    interface ITextStructureNavigator with 
+        member __.ContentType = _contentType
+        member __.GetExtentOfWord point = 

```

Comment:
Correct. The `__` is the idiomatic way of stating that `this` is unused in the member. At least that is how the F# code base approaches the problem and generally the community. There is actually an F# language proposal to make the `this` identifier optional on members to avoid this.

> https://github.com/fsharp/fslang-suggestions/issues/626

That being said it's the first time I've used it in the VsVim code base. At one point during this change I hit a place where I had two `x` identifiers and I wanted to use the outer only. Hence I ended up using `__` to see how it felt. Eventually I refactored out to this where it's not necessary. Could move it back to `x` as that's more consistent with our coding style.